### PR TITLE
DOCS-3151: Add env variable options for preinstall script

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/how-tos/provision-setup.md [hunk 1]
```diff
@@ -205,7 +205,18 @@ sudo ./preinstall.sh
...
 Follow the instructions.
-If you created a <FILE>viam-provisioning.json</FILE>, specify its location when prompted.
+If you created a <FILE>viam-provisioning.json</FILE>, specify its location as an environment variable or when prompted.
+
+You can set optional arguments by defining the following environment variables:
+
+<!-- prettier-ignore -->
+| Argument | Description |
+| -------- | ----------- |
+| `VIAM_JSON_PATH` | The path to the machine credentials <FILE>viam.json</FILE> file to be copied to the machine. |
```
Comment (Otterverse) [inline docs/how-tos/provision-setup.md:215]: It'll prompt for this one as well.

File: docs/how-tos/provision-setup.md [hunk 2]
```diff
@@ -205,7 +205,18 @@ sudo ./preinstall.sh
...
+| `PROVISIONING_PATH` | The path to the <FILE>viam-provisioning.json</FILE> file. The script will also prompt you for this file if not provided. |
+| `VIAM_AGENT` | The path to a beta or local build of `viam-agent`. Used for testing. |
```
Comment (Otterverse) [inline docs/how-tos/provision-setup.md:217]:
```suggestion
| `VIAM_AGENT_PATH` | The path to a beta or local build of `viam-agent`. Used for testing. |
```

**Review by Otterverse — CHANGES_REQUESTED** [review]
One variable name is wrong.

File: docs/how-tos/provision-setup.md (hunk 1 above)
Comment (npentrel) [inline docs/how-tos/provision-setup.md:215]:
```suggestion
| `VIAM_JSON_PATH` | The path to the machine credentials <FILE>viam.json</FILE> file to be copied to the machine. The script will also prompt you for this file if not provided. |
```

**Comment by npentrel** [comment]
Thanks @Otterverse !

**Review by Otterverse — APPROVED** [review]
LGTM

**Pull request merged**